### ISSUE TITLE
Uncaught List did not contain any matching elements

### ISSUE THREAD

**Issue by darting** [ionide/ionide-fsharp#149]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.0.19
**System**: Mac OS X 10.10.5
**Thrown From**: [ionide-fsharp](https://github.com/ionide/ionide-fsharp) package, v1.3.1

### Stack Trace

Uncaught List did not contain any matching elements

```
At /Users/z/.atom/packages/ionide-fsharp/lib/fsharp.js:2442

undefined
```

### Commands

```
  7x -1:20.6.0 core:select-down (atom-text-editor.editor.is-focused)
     -1:19.5.0 core:backspace (atom-text-editor.editor.is-focused)
     -1:19.2.0 core:save (atom-text-editor.editor.is-focused)
     -1:18.1.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -1:18.1.0 core:cancel (atom-text-editor.editor.is-focused)
     -0:56.2.0 snippets:next-tab-stop (atom-text-editor.editor.is-focused)
     -0:56.2.0 snippets:expand (atom-text-editor.editor.is-focused)
     -0:56.2.0 editor:indent (atom-text-editor.editor.is-focused)
  2x -0:55.2.0 core:backspace (atom-text-editor.editor.is-focused)
  2x -0:52.5.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:47.3.0 snippets:next-tab-stop (atom-text-editor.editor.is-focused)
     -0:47.3.0 snippets:expand (atom-text-editor.editor.is-focused)
     -0:47.3.0 editor:indent (atom-text-editor.editor.is-focused)
  7x -0:44.4.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:41.6.0 core:save (atom-text-editor.editor.is-focused)
     -0:04.7.0 autocomplete-plus:confirm (atom-text-editor.editor.is-focused.autocomplete-active)
```

### Config

```json
{
  "core": {
    "projectHome": "/Users/z/projects",
    "audioBeep": false,
    "disabledPackages": [
      "markdown-preview"
    ],
    "themes": [
      "atom-dark-ui",
      "one-dark-syntax"
    ]
  },
  "ionide-fsharp": {
    "ShowUseHighlights": false
  }
}
```

### Installed Packages

```coffee
# User
.bin, vundefined
atom-typescript, v7.0.1
atom-yeoman, v0.3.14
autocomplete-haskell, v0.5.1
file-icons, v1.6.9
haskell-ghc-mod, v0.9.12
ide-haskell, v0.7.2
ionide-fake, v1.2.0
ionide-fsharp, v1.3.1
ionide-fsi, v1.0.7
ionide-installer, v1.2.1
ionide-paket, v2.1.0
ionide-yeoman, v1.0.1
language-cmake, v0.1.4
language-haskell, v1.4.9
language-lua, v0.9.4
language-rust, v0.4.5
linter, v1.6.0
markdown-preview-plus, v2.1.1

# Dev
No dev packages
```

**Comment by vilinski** [comment]
[Enter steps to reproduce below:]

1. create an fsx file
2. write following code:

  open System.Data.SqlClient
  let bla() =
    use connection = new SqlConnection (connectionString)
    connection. <--- **crash on intellisense here**

**Atom Version**: 1.0.19
**System**: NUED-54438
**Thrown From**: [ionide-fsharp](https://github.com/ionide/ionide-fsharp) package, v1.3.1


### Stack Trace

Uncaught List did not contain any matching elements

```
At C:\Users\asvili\.atom\packages\ionide-fsharp\lib\fsharp.js:2442

undefined
```

### Commands

```
     -1:16 core:cancel (atom-text-editor.editor.is-focused)
     -0:25.3.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:19.9.0 core:move-down (atom-text-editor.editor.is-focused.autocomplete-active)
     -0:19.7.0 editor:move-to-first-character-of-line (atom-text-editor.editor.is-focused.autocomplete-active)
     -0:19.6.0 snippets:next-tab-stop (atom-text-editor.editor.is-focused)
     -0:19.6.0 snippets:expand (atom-text-editor.editor.is-focused)
     -0:19.6.0 editor:indent (atom-text-editor.editor.is-focused)
     -0:18.7.0 core:backspace (atom-text-editor.editor.is-focused)
     -0:17.3.0 core:move-down (atom-text-editor.editor.is-focused)
     -0:16.8.0 snippets:next-tab-stop (atom-text-editor.editor.is-focused)
     -0:16.8.0 snippets:expand (atom-text-editor.editor.is-focused)
     -0:16.8.0 editor:indent (atom-text-editor.editor.is-focused)
     -0:11.7.0 editor:move-to-end-of-screen-line (atom-text-editor.editor.is-focused)
     -0:10.9.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:05.8.0 autocomplete-plus:confirm (atom-text-editor.editor.is-focused.autocomplete-active)
  2x -0:04.0 core:backspace (atom-text-editor.editor.is-focused.autocomplete-active)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
atom-yeoman, v0.3.15
ionide-fake, v1.2.0
ionide-fsharp, v1.3.1
ionide-fsi, v1.0.7
ionide-installer, v1.3.0
ionide-paket, v2.1.0
ionide-webview, v1.0.0
ionide-yeoman, v1.0.1
linter, v1.8.1

# Dev
No dev packages
```

**Comment by Krzysztof-Cieslak** [comment]
Should be fixed with `1.3.3`
